care/emr/api/viewsets/patient.py

```
@@ -100,11 +106,60 @@ def get_queryset(self):
+                        patient=instance,
+                        config=get_object_or_404(
+                            PatientIdentifierConfig, external_id=identifier.config
+                        ),
+                    )
+                if identifier.value:
+                    identifier_obj.value = identifier.value
+                    identifier_obj.save()
+            instance.build_instance_identifiers()
+            instance.save()
+
     class SearchRequestSpec(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)